docs/csharp/whats-new/csharp-9.md

```diff
@@ -42,7 +42,6 @@ The record definition creates a `Person` type that contains two readonly propert
 - Override for <xref:System.Object.GetHashCode>
 - Copy and Clone members
 - `PrintMembers` and <xref:System.Object.ToString>
-- `Deconstruct` method
```

Comment:
We should probably add somewhere that an auto-generated `Deconstruct` method exists when declaring a `record` type with the following syntax:

```csharp
public record Person(string FirstName, string LastName);
```

But not when using the verbose syntax:

```csharp
public record Person
{
    ...
}
```

Reply:
I do call that out a couple paragraphs down, where I introduce positional records.